app/Model/Candidate/CandidateService.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Model\Candidate;
+
+use eGen\MessageBus\Bus\CommandBus;
+use Model\Candidate\Commands\SaveCandidates;
+
+final class CandidateService
+{
+    private CommandBus $commandBus;
+
+    private int $congressEventId;
+
+    public function __construct(
+        int $congressEventId,
+        CommandBus $commandBus
+    ) {
+        $this->congressEventId = $congressEventId;
+        $this->commandBus      =$commandBus;
+    }
+
+    public function saveCandidates() : void
+    {
+        $this->commandBus->handle(new SaveCandidates($this->congressEventId));
```

Comment:
Proč potřebujeme `CandidateService`, když ten command můžeme posílat přímo z presenteru?

Reply:
to co jsem ti včera psal, ze mi nejde dostat `$congressEventId` přímo do handleru